BP/common/landed_titles/BP_empire_roman.txt

```diff
@@ -0,0 +1,9045 @@
+	capital = c_ravenna
+	
+	definite_form = yes
+	
+	#Correct Title for Latins
+	ai_primary_priority = {
+		if = {
+			limit = { culture_group = culture_group:latin_group }
+			add = @correct_culture_primary_score
+		}
+	}
+	
+	cultural_names = {
+		#Roman Cultures
+		roman = cn_imperium_romanum_occidentale
+		romano_gallic = cn_imperium_romanum_occidentale
+		romano_hispanic = cn_imperium_romanum_occidentale
+		romano_british = cn_imperium_romanum_occidentale
+		african_romance = cn_imperium_romanum_occidentale
+		raetoroman = cn_imperium_romanum_occidentale
+		thraco_roman = cn_imperium_romanum_occidentale
+		illyro_roman = cn_imperium_romanum_occidentale
+		pannonian = cn_imperium_romanum_occidentale
+		dalmatian = cn_imperium_romanum_occidentale
+		noricroman = cn_imperium_romanum_occidentale
+		romano_gothic = cn_imperium_romanum_occidentale
+		romano_dacian = cn_imperium_romanum_occidentale
+		romano_frank = cn_imperium_romanum_occidentale
+		romano_aramean = cn_imperium_romanum_occidentale
+		romano_vandalic = cn_imperium_romanum_occidentale
+		#Byzantine Cultures
+		greek = cn_dytiki_romaiki_aftokratoria
+		aeolian = cn_dytiki_romaiki_aftokratoria
+		helleno_aramean = cn_dytiki_romaiki_aftokratoria
+		greek_gothic = cn_dytiki_romaiki_aftokratoria
+		greek_persian = cn_dytiki_romaiki_aftokratoria
+		isaurian_persian = cn_dytiki_romaiki_aftokratoria
+		grecco_axumite = cn_dytiki_romaiki_aftokratoria
+		bosporan = cn_dytiki_romaiki_aftokratoria
+		pontic = cn_dytiki_romaiki_aftokratoria
+		tsakonian = cn_dytiki_romaiki_aftokratoria
+		massalian = cn_dytiki_romaiki_aftokratoria
+		helleno_thracian = cn_dytiki_romaiki_aftokratoria
+	}
+	
+	#Should be creatable by decision
+	can_create = { always = no }
+}
+
+e_italy = {
+	color={ 178 34 34 }
+	color2={ 255 255 255 }
+	capital = c_lombardia
+	
+	cultural_names = {
+		#Roman Cultures
+		roman = cn_italia
+		romano_gallic = cn_italia
+		romano_hispanic = cn_italia
+		romano_british = cn_italia
+		african_romance = cn_italia
+		raetoroman = cn_italia
+		thraco_roman = cn_italia
+		illyro_roman = cn_italia
+		pannonian = cn_italia
+		dalmatian = cn_italia
+		noricroman = cn_italia
+		romano_gothic = cn_italia
+		romano_dacian = cn_italia
+		romano_frank = cn_italia
+		romano_aramean = cn_italia
+		romano_vandalic = cn_italia
+		#Byzantine Cultures
+		greek = cn_italia
+		aeolian = cn_italia
+		helleno_aramean = cn_italia
+		greek_gothic = cn_italia
+		greek_persian = cn_italia
+		isaurian_persian = cn_italia
+		grecco_axumite = cn_italia
+		bosporan = cn_italia
+		pontic = cn_italia
+		tsakonian = cn_italia
+		massalian = cn_italia
+		helleno_thracian = cn_italia
+	}
+	
+	#Correct title for Italians
+	ai_primary_priority = {
+		if = {
+			limit = { culture = culture:italian }
+			add = @correct_culture_primary_score
+		}
+	}
+	
+	k_annonaria = {
+		color={ 220 20 60 }
+		capital = c_ravenna
+		
+		#Keep the ostrogoths et al. from changing away from k_italy
+		can_create = {
+			Neither Goths, and 
```

Comment:
This should probably be commented out or removed

Reply:
Removed that error.